prometheus_hardware_exporter/collector.py

```diff
@@ -902,7 +1052,174 @@ def fetch(self) -> List[Payload]:
                         },
                     )
                 )
+        return payloads
+
+    def _create_processor_metric_payload(
```

Comment:
**nitpick:** 

The doc-string practice is not very good for those `_create_xxx` functions. It's worth to provide some example if the struct is complex.

Reply:
Agree with you here. I'll add some helpful docstrings :+1: